examples/p4est_2d_dgsem/elixir_euler_airfoil_mach2.jl

```diff
@@ -0,0 +1,104 @@
+
```

Comment:
It might be worthwhile to mention in a comment near the mesh construction and/or change the file name to indicate that this is the specific NACA6412 airfoil. Also adding a link or so to clarify what this airfoil is or how the boundary data and/or mesh was generated would be helpful.

Reply:
Totally agreed, I will also upload the `.geo` input file to `gmsh` and also plan to write a tutorial on this in the documentation.

Reply:
Great, thanks! Also, once the `.geo` data is available I would like to try HOHQMesh to create the NACA6412 airfoil :)

Reply:
You can already have a look:

https://gist.github.com/DanielDoehring/5ade6d93629f0d8c23a598812dbee2a9

Reply:
Thanks! What angle of attack did you use for this data?

Reply:
I took the setup from 

https://github.com/cfsengineering/GMSH-Airfoil-2D

Reply:
Okay, that is what I thought. Here is a first attempt from the HOHQMesh side of things
<img width="695" alt="Screenshot 2024-01-10 at 00 17 56" src="https://github.com/trixi-framework/Trixi.jl/assets/25242486/40a07512-d3cf-472f-82ac-67eee829019c">